bin/validate-commits.js

```diff
@@ -64,7 +73,6 @@ function getBranch() {
   if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST !== 'false') {
     return {
       from: process.env.TRAVIS_BRANCH,
-      to: process.env.TRAVIS_BRANCH,
```

Comment:
This came from `${TRAVIS_BRANCH}..`, I'm guessing it is the `from` part 🤷 

definitely not `from` and `to`